exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -649,13 +648,26 @@ func (b *Bitfinex) SubmitOrder(ctx context.Context, o *order.Submit) (*order.Sub
 	var orderID string
 	status := order.New
 	if b.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
-		orderID, err = b.WsNewOrder(&WsNewOrderRequest{
+		symbolStr, err := b.fixCasing(fPair, o.AssetType) //nolint:govet // intentional shadow of err
+		if err != nil {
+			return nil, err
+		}
+		orderType := strings.ToUpper(o.Type.String())
+		if o.AssetType == asset.Spot {
+			orderType = "EXCHANGE " + orderType
+		}
+		req := &WsNewOrderRequest{
 			CustomID: b.Websocket.AuthConn.GenerateMessageID(false),
```

Comment:
Suggestion: This is not your doing but CustomID is also set in `WSNewOrder` so this can be either removed here or removed in that function and an error returned?

Reply:
Agreed.